src/platform/GenericOTARequestorDriver.cpp

```diff
@@ -21,11 +21,22 @@
 #include <platform/OTAImageProcessor.h>
 #include <platform/OTARequestorInterface.h>
 
+// This implementation of the OTARequestorDriver operates according the following rules:
+// - Only a single timer can be active at any given moment
```

Comment:
Is this rule really necesary? It should be explained why, since the API doesn't make it seem at all like there can only be one active at a time.

Reply:
What I am trying to convey here is that this is a particular implementation of OTARequestorDriver class. It made certain choices and follows self-imposed conventions (like one active timer at a time) but none of this is a requirement.

Reply:
Updated wording